server/controllers/users.js

```diff
@@ -1,10 +1,9 @@
-import { db } from "../database";
 import bcrypt from "bcryptjs";
```

Comment:
Line 72 of this file is sending a wrong error message. Please correct this

Reply:
Okay... on it